IdentityCore/IdentityCore.xcodeproj/project.pbxproj

```diff
@@ -6041,7 +6041,9 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D6CF4EA01FC3626C00CD70C5 /* identitycore__idlib__mac.xcconfig */;
 			buildSettings = {
+				CLANG_WARN_SEMICOLON_BEFORE_METHOD_BODY = YES;
```

Comment:
Should we put these settings into config files instead? You can then include our config file in your config file to ensure same settings are used for both of us.

Reply:
Ours are actually set by CMake, using `-W...` flags rather than config options, so including a config file would be difficult on our end.

For now I think I'm okay with just adding these options here - so far these two have really been the only ones to cause problems.

Reply:
Makes sense. But we still store all our settings in config files, so even if you don't include it in your project, I'd prefer you to apply these changes in our config file :)

Reply:
Sure!